scripts/ts-bin/pair_comments_for_relativerating.ts

```diff
@@ -19,158 +19,170 @@ Usage: Convert answers with multiple parts into separate rows.
   ts-node ./ts-bin/pair_comments_for_relativerating.ts \
     --infile="./tmp/path-to-input.jsonl" \
     --output_path_prefix="./tmp/path-to-paired"
+
+  Once this has been done, the jsonlines can be converted to CSV file using:
+  ./node_modules/.bin/json2csv --ndjson \
+    --input="./tmp/path-to-paired.jsonl" \
+    --output="./tmp/path-to-paired.csv"
 */
 
-import * as yargs from 'yargs';
 import * as fs from 'fs';
 import * as readline from 'readline';
-// import * as jsonexport from "jsonexport/dist"
-// import * as stream from 'stream';
+import * as yargs from 'yargs';
 
 // Command line arguments.
 interface Params {
-  infile:string,
-  output_path_prefix:string,
-  output_path_postfix:string,
-  left_id_prefix:string,
-  right_id_prefix:string,
-  left_text_prefix:string,
-  right_text_prefix:string,
-  entries_per_row:number,
-  rows_per_file:number,
-};
+  infile: string, output_path_prefix: string, output_path_postfix: string,
+      left_id_prefix: string, right_id_prefix: string, left_text_prefix: string,
+      right_text_prefix: string, entries_per_row: number, rows_per_file: number,
+}
+;
 
 interface InputObj {
-  comment_text:string;
-  id:string;
+  comment_text: string;
+  id: string;
+}
+
+// A string -> string table that will be outputed as JSON in a file.
+type OutFileData = {
+  [key: string]: string
+}[];
+
+function createAllPairsFiles(data: InputObj[]): OutFileData[] {
+  let out_files: OutFileData[] = [];
+  let pair_id = 1;
+  let pair_id_postfix = '';
+  let out_pairs_for_row: {[key: string]: string} = {};
+  let out_rows_for_file: {[key: string]: string}[] = [];
```

Comment:
You can use the new OutFileData type here.

Reply:
oh yeah, thanks!

Reply:
done

Reply:
done